### ISSUE TITLE
Does it work in dynamic scenarios? 

### ISSUE THREAD

**Issue by hzfGuardian** [jiaxiZeng/Temporally-Consistent-Stereo-Matching#6]
As shown in your paper, you use relative poses among temporal frames. Does it based on an assumption that the scene must have no moving objects?

**Comment by jiaxiZeng** [comment]
It works in dynamic scenarios, the pose-based projection is only used to provide a prior for the current frame. The refinement module will correct the error from the object motion by matching the left and right images.  In addition, since our model can run at nearly real-time speed (10fps), the movement of dynamic objects can be considered minimal, so the impact of this prior bias is not significant. Specifically, you can refer to our results on the KITTI dataset, which include examples of disparity prediction for dynamic objects.

**Comment by hzfGuardian** [comment]
@jiaxiZeng Thanks for your reply. Furthermore, for a real video sequence where the camera doesn't move but the objects move, can I set param['T'] and param['previous_T'] both as np.eye(4), as you mentioned ?

**Comment by jiaxiZeng** [comment]
@hzfGuardian  Yes, you can. I suggest that in addition to using the Tartanair checkpoint, you also try the SceneFlow checkpoint, as Tartanair does not contain many dynamic objects. Alternatively, you could fine-tune on your own dataset.
